my-dsl.md

```diff
@@ -2,37 +2,46 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+I found the DSL [BUGS.](http://fisher.utstat.toronto.edu/pub/escobar/bug5/doc/manual05.ps)
 
 # Domain
 _Describe the language's domain in five words._
 
+"Bayesian interference on statistical problems" - [BUGS Manual, Section 1.1](http://fisher.utstat.toronto.edu/pub/escobar/bug5/doc/manual05.ps).
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+The model runs several hundred times to find an estimate for the unknown quanities in your model and then uses them to run the number of requested simulations of the test, tracking the parameters of interest. It returns a file containing the series of values simulated for the users parameter of interest.
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+My DSL falls very close to the "purely" domain-specific end of the spectrum. It is not a Turing complete language according to [this article](http://fisher.utstat.toronto.edu/pub/escobar/bug5/doc/manual05.ps). It has a clear domain focus on complex statistical modeling. It has the fluency of a langauge since you can write an entire block of code in it. BUGS is clearly intended for human statisticians to instruct a computer to do calculations for them so it is a computer programming language. While BUGS does include for loops and arrays which are unusual in a DSL, these are included purely to allow clear organization of large quantities of data used in and generated by statistical models. For loops are used to describe large, repetitive data structure clearly and succinctly. According to the BUGS manual, for loops "constitute a declarative description of the graphical model, and should not be interpreted in a procedural way"
 
```

Comment:
This is interesting, I think that my perception of this scale is a bit different. I would put this language a bit closer to the middle of the spectrum. Even though the language features such as for loops, arrays, etc are included with that specific statistical purpose in mind, I wonder about the extent of their capabilities. I like how you included the BUGS manual quotation, since that reinforces the domain-specific argument, though I would be curious to play around with the language more extensively and try to implement different functionality with the existing loop and array settings.